goals/operationalUsecases.md

```diff
@@ -0,0 +1,59 @@
+1. ## **Managing Users**
+    **1.1 Terms Acceptance**
```

Comment:
suggest using the standard numbered list format for markdown
1.
   1.
   1.
1.